### DISCUSSION TITLE
Rename "indirect expand" to "reverse expand" in documentation

### DISCUSSION THREAD

**Discussion by lbensaad** [pocketbase/pocketbase#3136] [Q&A]
The `indirect relation` term is confusing. In general  if A relates to B and B relates to C we say that A has an indirect relation to C. But we can say that B has a reverse relation to A, and that is what we want. 
I suggest calling it `reverse relation`

**Comment by ganigeorgiev** [comment]
Hm, I'm not really sure, maybe you are right.

I'm using "indirect relation" because in our case it indicates that the relation happens though a field from another collection (aka. the notation resolution flow is _main->other->field_) and it just seems more correct to me, but now that you mentioned "reverse relation" I'm not really sure anymore.

I'll leave it aside for now to think a little more on it and will eventually update it together with some other planned relation changes for the next release.

**Comment by lbensaad** [comment]
Thanks for considering it. in your example here we have only two nodes, `main` and `other` and they are of the same type (collection in our case). `field` in your example is not a node is a property of the node (collection) `other`.
In graph theory, we talk about indirect relation when we have at least 3 nodes as I mentioned before, A, B and C.
When we have 2 nodes (2 collections in our case) then we are talking about reverse relation or back-relation.

**Reply by ganigeorgiev** [reply]
I think the main reason why I decided on "indirect" was that I was looking for an antonym of the "direct" relation.

I could see why you may think it is confusing as googling for it seems that "indirect" is commonly used also as synonym for "transitive", which for some reason didn't cross my mind when writing the docs.

As mentioned earlier, I will consider eventually replacing it but I'll have to research it better because as of now "reverse relation" sounds a little confusing to me, especially in the context of expand, but again that's maybe just me (_"back relation" is somewhat less ambiguous_).

**Comment by lbensaad** [comment]
In entgo, they use the terms `owner/inverse`  and also call the indirect relation  `back-relation` , see it [here](https://entgo.io/docs/schema-edges/):

**Comment by lbensaad** [comment]
I think the most natural terms are `back-relation` and `back-expand`
